src/components/Modals/Send/utils.ts

```diff
@@ -155,12 +155,12 @@ export const handleSend = async ({
         const contractAddress = tokenOrUndefined(fromAssetId(asset.assetId).assetReference)
         const { accountNumber } = bip44Params
         return await (adapter as unknown as EvmBaseAdapter<EvmChainId>).buildSendTransaction({
-          memo,
```

Comment:
This was rugged, the only reason this was actually working was that it didn't have to - this is only used for THOR deposit/withdraw savers native/token *sends* (vs. contract calls), which don't require any memo since only the dust amount being valid is used to identify the address, and whether it is a deposit/withdraw.

For akschual contract calls on EVM chains, we use `buildAndBroadcast`, not `handleSend`.